src/main/java/core/basesyntax/service/impl/AddOperation.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+import java.util.Optional;
+
+public class AddOperation implements FruitOperationHandler {
+    private int newBalance;
```

Comment:
but in here it should local variable, you need to refresh it each method call anyway

Reply:
Done.